src/Sarif.Driver.UnitTests/Sdk/EntryPointUtilitiesTests.cs

```diff
@@ -0,0 +1,154 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text.RegularExpressions;
+
+using FluentAssertions;
+
+using Moq;
```

Comment:
Extra blank line. Here we have three different third-party usings, and only one is separated.